libsodium-ios/include/sodium/version.h

```diff
@@ -0,0 +1,33 @@
+
+#ifndef sodium_version_H
+#define sodium_version_H
+
+#include "export.h"
+
+#define SODIUM_VERSION_STRING "1.0.16"
+
+#define SODIUM_LIBRARY_VERSION_MAJOR 10
+#define SODIUM_LIBRARY_VERSION_MINOR 1
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+SODIUM_EXPORT
+const char *sodium_version_string(void);
+
+SODIUM_EXPORT
+int         sodium_library_version_major(void);
+
+SODIUM_EXPORT
+int         sodium_library_version_minor(void);
+
+SODIUM_EXPORT
+int         sodium_library_minimal(void);
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif
```

Comment:
why do you include libsodium.a? isn't it better to build it?

Reply:
No, because we need to adopt c source code for iOS. It is easier to build static library once. In the end of the day only part related to scrypt will be included to IrohaCrypto.framework at build time. So we just waste some not significant space of git repository.

Reply:
If all iOS devices can successfully consume this libsodium.a, then it is ok